src/routes/news/containers/LocalNewsPage.js

```diff
@@ -49,12 +49,11 @@ export class LocalNewsPage extends React.Component<PropsType> {
     }
 
     const currentCity: CityModel = cities && cities.find(cityElement => cityElement.code === city)
-
     if (!currentCity.pushNotificationsEnabled) {
-      const type = currentCity.tunewsEnabled ? 'tunewsItem' : 'category'
-      const error = new ContentNotFoundError({ type, id: path, city: city, language })
+      const error = new ContentNotFoundError({ type: 'category', id: path, city: city, language })
```

Comment:
Why are we using here 'category' instead of 'tunews'?

Reply:
Imo it should be `category` all the time. If a feature is disabled, we want to show a generic error (which is category in our case). I think the same applies for the questions below.

Reply:
Yep, @klinzo is right. Changing this was the main goal of this task.